fastpack/cli.py

```diff
@@ -4,13 +4,17 @@
 import sys
 from pathlib import Path
 
+# S603 `subprocess` call: check for execution of untrusted input
+# ruff: noqa: S603
+# S607 Starting a process with a partial executable path
+# ruff: noqa: S607
+
 TEMPLATE_DIR = Path(__file__).parent / "templates"
 PYPROJECT_PATH = TEMPLATE_DIR / "pyproject.toml"
 
 
 def main():
-
-    with open(PYPROJECT_PATH) as f:
+    with Path.open(PYPROJECT_PATH) as f:
```

Comment:
This is a general convention question that doesn't necessarily require action.  I have conventions mixed across a lot of my packages.  Is `Path.open(PYPROJECT_PATH) as f:` or `PYPROJECT_PATH.open() as f:` the right "pythonic" approach?